doc/subsystems/networking/usbnet_setup.rst

```diff
@@ -0,0 +1,100 @@
+.. _usb_device_networking_setup:
+
+USB Device Networking
+#####################
+
+This page describes how to set up networking between a Linux host
+and a Zephyr application running on USB supported devices.
+
+The board is connected to Linux host using USB cable
+and provides an Ethernet interface to the host.
+The :ref:`echo-server-sample` application from the Zephyr source
+distribution is run on supported board.  The board is connected to a
+Linux host using a USB cable providing an Ethernet interface to the host.
+
+Basic Setup
+***********
+
+To communicate with the Zephyr application over a newly created Ethernet
+interface, we need to assign IP addresses and set up a routing table for
+the Linux host.
+After plugging a USB cable from the board to the Linux host, the
+``cdc_ether`` driver registers a new Ethernet device with a provided MAC
+address.
+
+You can check that network device is created and MAC address assigned by
+running dmesg from the Linux host.
+
+.. code-block:: console
+
+   cdc_ether 1-2.7:1.0 eth0: register 'cdc_ether' at usb-0000:00:01.2-2.7, CDC Ethernet Device, 00:00:5e:00:53:01
+
+We need to set it up and assign IP addresses as explained in the following
+section.
+
+Choosing IP addresses
+=====================
+
+To establish network connection to the board we need to choose IP address
+for the interface on the Linux host.
+
+It make sense to choose addresses in the same subnet we have in Zephyr
+application. IP addresses usually set in the project configuration files
+and may be checked also from the shell with following commands. It is
+required to connect to the board with serial console and run commands:
+
+.. code-block:: console
+
+   shell> net iface
+
+   Interface 0xa800e580 (Ethernet)
+   ===============================
+   Link addr : 00:00:5E:00:53:00
+   MTU       : 1500
+   IPv6 unicast addresses (max 2):
+           fe80::200:5eff:fe00:5300 autoconf preferred infinite
+           2001:db8::1 manual preferred infinite
+   ...
+   IPv4 unicast addresses (max 1):
+           192.0.2.1 manual preferred infinite
+
+So we found that board has allocated one IPv4 address two IPv6 addresses
```

Comment:
How about:

    This command shows that one IPv4 address and two IPv6 addresses have been assigned to the board.

Reply:
Ok